lib/SPIRV/OCL20ToSPIRV.cpp

```diff
@@ -509,7 +509,7 @@ OCL20ToSPIRV::visitCallNDRange(CallInst *CI,
     default:
       assert(0 && "Invalid number of arguments");
     }
-    return getSPIRVFuncName(OpBuildNDRange);
+    return getSPIRVFuncName(OpBuildNDRange, "_" + std::to_string(Len) + "D");

```

Comment:
I don't understand this change. According to description it's done to avoid some collision. There should be no collisions - all the OpenCL built-ins must map to the same "LLVM-SPIR-V" built-in __spirv_BuildNDRange_*.